README.md

```diff
@@ -96,7 +96,7 @@ for additional information.
     [CMake](https://cmake.org/download/) >= 3.1.3
   * We do not use CMake to build any C++ code, but to copy files and run
     **Ant**.
-* **C++ compiler**: [Visual Studio 2015](https://www.visualstudio.com/)
+* **C++ compiler**: [Visual Studio 2017](https://www.visualstudio.com/)
```

Comment:
This is fine but I bet the default download is now VS 2019.

Reply:
Thanks @chrisdembia True default is 2019 but they still have links for older downloads. I don't know if we can be sure the code compiles on vs2015 when no one on the dev team is using it so I used the defacto minimum maintained version.